signature-specification.md

```diff
@@ -1,243 +1,118 @@
-1. Descriptor MAY contain `artifactType` field for artifact manifests, or the `config.mediaType` for `oci.image` based manifests.
+- `subject` : Required property whose value is the **OCI descriptor** of the subject manifest that is being signed, with media type of `application/vnd.cncf.oras.artifact.descriptor.v1+json`.
+  - Descriptor MUST contain `mediaType`, `digest`, `size` fields.
+  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. Notary v2 uses annotations for storing both Notary specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary v2 and MUST NOT be used outside this specification.
+  - Descriptor MAY contain `artifactType` field for artifact manifests, or the `config.mediaType` for `oci.image` based manifests.
 
-Examples:
+#### Examples
 
 ```jsonc
 {
-   "mediaType": "application/vnd.oci.image.manifest.v1+json",
-   "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
-   "size": 16724,
-   "annotations": {
-      "io.wabbit-networks.buildId": "123"  // user defined signed attribute.
-   }
+  "subject": {
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
+    "size": 16724,
+    "annotations": {
+        "io.wabbit-networks.buildId": "123"  // user defined metadata
+    }
+  }
 }
 ```
 
-```json
+```jsonc
 {
-   "mediaType": "sbom/example",
-   "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
-   "size": 32654
+  "subject": {
+    "mediaType": "sbom/example",
+    "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+    "size": 32654
+  }
 }
 ```
 
 ### Signed Attributes
 
-Notary v2 requires the signature envelope to support the following signed attributes:
-
-- **Creation time**: The time at which the signature was generated.
-  Its value should be numeric representing the number of seconds (not milliseconds) since Epoch.
-- **Expiration time**: The time after which signature shouldn't be considered valid.
-  Its value should be numeric representing the number of seconds since Epoch.
-  This is an OPTIONAL attribute.
+Signed attributes/claims are additional metadata apart from the payload, which are required to support the signature verification process.
 
-### Unsigned Attributes
+- Any claims MUST NOT be stored/appended in the payload itself, as the payload is only parsed and processed once the signature has been verified (signature is valid, and from a trusted key) and trust is established.
+- Specific claims can be either required or optional.
+- Claims that if present, MUST be processed by a verifier MUST be marked as critical.
```

Comment:
This sentence is ambiguous for me. Adding a pair of brackets enables me understand it faster:
```
Claims (that if present, MUST be processed by a verifier) MUST be marked as critical.
```
Request for rewording.

Reply:
How about
> Claims that are present and MUST be processed by a verifier MUST be marked as critical.

Reply:
Updated to

>Claims that MUST be processed by a verifier MUST be marked as critical. Some claims may be optional and critical, i.e. they MUST be processed by a verifier only if they are present.